Resources/skeleton/demo-bundle/DemoBundle/Resources/config/services.xml

```diff
@@ -5,14 +5,14 @@
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <services>
-        <service id="twig.extension.acme.demo" class="Acme\DemoBundle\Twig\Extension\DemoExtension" public="false">
+        <service id="twig_extension_demo" class="DemoBundle\Twig\Extension\DemoExtension" public="false">
             <tag name="twig.extension" />
             <argument type="service" id="twig.loader" />
         </service>
 
-        <service id="acme.demo.listener" class="Acme\DemoBundle\EventListener\ControllerListener">
+        <service id="listener_demo" class="DemoBundle\EventListener\ControllerListener">
```

Comment:
this does not follow the best practice. We keep the ``app.`` prefix to avoid conflicts with services added in Symfony in future versions (otherwise we would have to consider new short names in Symfony as BC breaks): http://symfony.com/doc/current/best_practices/business-logic.html#services-naming-and-format